articles/cognitive-services/text-analytics/how-tos/text-analytics-how-to-process-offsets.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Process offsets in API output
+titleSuffix: Azure Cognitive Services
+description: This article explains how to process text offsets in the output of the Azure Cognitive Services Text Analytics APIs.
+services: cognitive-services
+author: jdesousa
+manager: andyh
+
+ms.service: cognitive-services
+ms.subservice: text-analytics
+ms.topic: conceptual
+ms.date: 02/27/2020
+ms.author: jdesousa
+---
+
+# How to process text offsets in the output of the Text Analytics API
+
+After [making a call](text-analytics-how-to-call-api.md) to Text Analytics, the JSON payload in the response includes elements that are specific to each of of the APIs.
+
+Whenever **offsets** are returned, e.g. for [Named Entity Recognition](text-analytics-how-to-entity-linking) or [Sentiment Analysis](text-analytics-how-to-sentiment-analysis), bear in mind that
+
++ HTTP POST/GET payloads are encoded in [UTF-8](https://www.w3schools.com/charsets/ref_html_utf8.asp), which may or may not be the default character encoding on your client-side compiler/operating system;
```

Comment:
Add a first bullet saying that different programming languages may have different ways to represent text. For example: Python3 users may be using Unicode based code points offsets, Java users may use UTF-16 and GO users may use UTF-8.